CONTRIBUTING.md

```diff
@@ -19,7 +19,11 @@ executable. No need to reinstall the app to test your changes.
 Running Unit Tests
 ------------------
 
-You are also able to run unit tests by executing `npm test`.
+The following dependencies are required to run tests:
+
+    $ npm install -g istanbul ytestrunner yuitest
+
+You are able to run unit tests by executing `npm test`.
```

Comment:
Why did you need to do this? I think `npm test` should works after run `npm install`.